libraries/api-extractor-model/src/items/ApiDeclaredItem.ts

```diff
@@ -74,6 +82,25 @@ export class ApiDeclaredItem extends ApiDocumentedItem {
     return this._excerptTokens;
   }
 
+  /**
+   * The file URL path relative to the `projectFolder` and `projectFolderURL` fields
+   * as defined in the `api-extractor.json` config. Is `undefined` if the path is
+   * the same as the parent API item's.
+   */
+  public get fileUrlPath(): string | undefined {
```

Comment:
Seems like this should be a property of the `SourceLocation` class. What do you think?

Reply:
I think I had that in a previous iteration, but the code seemed cleaner with the property on `ApiDeclaredItem`. There are already matching properties on `IApiDeclaredItemOptions` and `IApiDeclaredItemJson`, as well as a matching private member on `ApiDeclaredItem`. The getter is also not particularly useful/interesting on its own (as it's undefined if the path is the same as the parent API item's path), and is just used internally by api-extractor-model to construct `SourceLocation` objects and serialize succinct `.api.json` files. It's really not intended to be used by consumers. It seemed like `ApiDeclaredItem` was a better place for such a member...